cmd/agent/main.go

```diff
@@ -9,12 +9,12 @@ import (
 	"github.com/kairos-io/kairos/internal/bus"
 	"github.com/kairos-io/kairos/internal/webui"
 
+	bundles "github.com/kairos-io/kairos-sdk/bundles"
```

Comment:
maybe we should have named the repo just "sdk" and make the golang files top level? so imports are cleaner:
```
import (
 "github.com/kairos-io/sdk/..."
)
```

Reply:
Thougth about that but feel like it was very...generic. Like if we had down the line a different sdk for something else? kairos-sdk looked like it was very clear what it is for. I dunno...we have time to change this, I have no strong feelings one way or the other